net/instaweb/rewriter/add_instrumentation_filter.cc

```diff
@@ -95,7 +101,12 @@ void AddInstrumentationFilter::AddHeadScript(HtmlElement* element) {
     added_head_script_ = true;
     // TODO(abliss): add an actual element instead, so other filters can
     // rewrite this JS
-    HtmlCharactersNode* script = driver()->NewCharactersNode(NULL, kHeadScript);
+    HtmlCharactersNode* script = NULL;
+    if(driver()->options()->Enabled(RewriteOptions::kPedantic)){
```

Comment:
We use a space between "if" and "(driver)", and before the "{"

https://google.github.io/styleguide/cppguide.html#Conditionals